src/s_sound.c

```diff
@@ -293,6 +213,13 @@ void S_StartSound(const mobj_t *origin, int sfx_id)
       I_Error("Bad sfx #: %d", sfx_id);
 #endif
 
+   // Assigns the handle to one of the channels in the mix/output buffer.
+   handle = I_GetChannel();
+   if(handle < 0)
```

Comment:
Please use `if (`, especially when adding new code. I know `if(` is the original style, but it's causing me aches. 😉

Reply:
Fixed by https://github.com/fabiangreffrath/woof/pull/1118/commits/92a1f239122ddde7e5a6602ebddc0a9e2be0c74f